Makefile.js

```diff
@@ -119,6 +120,9 @@ function release(type) {
     echo("Generating changelog");
     target.changelog();
 
+    // capture changelog
+    changes = execSilent("git diff CHANGELOG.md");
```

Comment:
This would also include text like this:

```
diff --git a/CHANGELOG.md b/CHANGELOG.md
index d9db605..a884b50 100644
--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,4 @@
```